docs/source/_templates/datasets/dataset.rst.j2

```diff
@@ -3,7 +3,15 @@
 {{ schema.description or '' }}
 
 * **ID:** {{ schema_name }}
-* **WFS:** {% if wfs_url %}{{ wfs_url }} (:doc:`documentatie <../wfs-datasets/{{ schema_name }}>`){% else %}N.B.{% endif %}
+* **VERSIE:** {{ schema.version }}
+* **ENDPOINTS:**
```

Comment:
Misschien als **Versie:** en **Endpoints:** opschrijven? Of zijn de HOOFDLETTERS juist wel fraai? :-)

Reply:
Ja goeie, vind Versie en Endpoints ook mooier, ik pas het aan